ajax/libs/overpass/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "overpass",
   "version": "3.0.2",
-  "filename": "overpass-webfont/overpass.css",
+  "filename": "overpass-webfont/overpass.min.css",
```

Comment:
@sufuf3 I did exactly that. However, I get a merge conflict after I edit the first commit.

I think that's because I change `overpass.css` to `overpass.min.css` in the second commit before.